lib/ddtrace/contrib/rack/middlewares.rb

```diff
@@ -32,19 +40,34 @@ def initialize(app, options = {})
           @app = app
         end
 
+        def compute_queueing_time(env, tracer)
+          if Datadog.configuration[:rack][:request_queuing]
```

Comment:
Same here about the early return comment

Reply:
👍